src/app/models/field/__tests__/emailField.spec.ts

```diff
@@ -66,8 +66,8 @@ describe('models.fields.emailField', () => {
     const expected = merge(EMAIL_FIELD_DEFAULTS, mockEmailField, {
       _id: expect.anything(),
       autoReplyOptions: {
-        // Regardless, should be false since ResponseMode is Encrypt.
-        includeFormSummary: false,
+        // Should be true now since pdf summary is supported on encrypt
```

Comment:
Can remove the comment, since we can assume that this is true without exception.